test/controllers/OpenPaymentsControllerSpec.scala

```diff
@@ -73,7 +73,7 @@ class OpenPaymentsControllerSpec extends ControllerBaseSpec {
 
           (mockPaymentsService.getOpenPayments(_: String)(_: HeaderCarrier, _: ExecutionContext))
             .expects(*, *, *)
-            .returns(Future.successful(Right(Payments(Seq(payment, payment)))))
+            .returns(Future.successful(Some(Payments(Seq(payment, payment)))))
```

Comment:
Can you remove the unused imports please and add types to results and documents throughout the file please.